discord/lib/handlers/authenticated.test.js

```diff
@@ -1,23 +1,27 @@
-jest.mock('../utils', () => {
-  const originalModule = jest.requireActual('../utils');
-  return {
-    ...originalModule,
-    // Mock authFromCode so we can spy on it.
-    authFromCode: jest.fn((_fetch, _data) => {
-      return Promise.resolve({ json: jest.fn().mockReturnValue({
-        access_token: 'some-access-token',
-        expires_in: 90,
-      })});
-    }),
-  };
-});
+// https://jestjs.io/docs/ecmascript-modules#differences-between-esm-and-commonjs
+import { jest } from '@jest/globals';
+
+// jest.mock('../utils', () => {
+//   // FIXME: how to jest.requireActual in esm?
+//   const originalModule = jest.requireActual('../utils');
+//   return {
+//     ...originalModule,
+//     // Mock authFromCode so we can spy on it.
+//     authFromCode: jest.fn((_fetch, _data) => {
+//       return Promise.resolve({ json: jest.fn().mockReturnValue({
+//         access_token: 'some-access-token',
+//         expires_in: 90,
+//       })});
+//     }),
+//   };
+// });
```

Comment:
TODO: port to ava

Reply:
https://github.com/testdouble/contributing-tests/wiki/Don't-mock-what-you-don't-own